### ISSUE TITLE
mocha.requires (settings.json) is not taken into consideration when debuging a single test

### ISSUE THREAD

**Issue by rgustavsson** [maty21/mocha-sidebar#55]
When I run all my tests through a debug configuration like this:

```
        {
            "type": "node",
            "request": "launch",
            "name": "Debug Mocha Tests",
            "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
            "args": [
                "-u",
                "tdd",
                "--timeout",
                "10000",
                "--colors",
                "${workspaceFolder}/Tests/Mocha/Features",
                "--require",
                "${workspaceFolder}/config/local.js"
            ],
            "internalConsoleOptions": "openOnSessionStart"
        }
```
**THEN** the --require option is taken into consideration and I can debug the code inside `config/local.js`.

**BUT** when I open a single test file and inside the test I click on "Debug Item", then the `config/local.js` is not loaded.
How to I get my --require files to load before the test when clicking on "Debug item"?

In .vscode I have a settings.json file that looks like this:

```
{
    "mocha.files.glob": "tests/mocha/features/*.js",
    "mocha.options": {
        "timeout" : 10000
    },
    "mocha.requires": [
        "config/local.js"
    ]
}
```

I've read through the Q&A without any luck.

Thanks

**Comment by yehiyam** [comment]
Hi
Could you try to require the file with ./config/local.js (relative path)

**Comment by rgustavsson** [comment]
Thanks 
I moved the local.js to the same folder as my test.js and changed the require to relative as per you request, and that worked (just ./local.js) .
I'd like to have it like this:
- config
-- local.js
-tests
-- myTest.js

but when I try with ./../config/local.js or ../config/local.js it doesn't work..

Do you know what Im doing wrong?

**Comment by rgustavsson** [comment]
I'm ok with having the config.js in my tests/ folder for now but it looks like it causes a conflict with discovering tests..

if I have it like this:
tests\
--test1.js
--config.js
--test2.js

then test2.js is not discovered by the sidebar, until I remove the config file (and do the same in workspace settings.json)

**Comment by yehiyam** [comment]
fixed it in v0.15.0.
